src/ActionMailerNext.Standalone/Helpers/HandlebarsFilesTemplateResolver.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace ActionMailerNext.Standalone.Helpers
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class HandlebarsFilesTemplateResolver : ITemplateResolver
+    {
+        private readonly string _viewPath;
+
+        /// <summary>
+        ///     Creates a template resolver using the specified path.  If no path is given, this defaults to "Views"
+        /// </summary>
+        /// <param name="viewPath">The path containing your views</param>
+        public HandlebarsFilesTemplateResolver(string viewPath)
+        {
+            _viewPath = viewPath;
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="name"></param>
+        /// <returns></returns>
+        public string Resolve(string name)
+        {
+            if (string.IsNullOrWhiteSpace(name))
```

Comment:
Don't we get a warning for "if without quotes" any more?

Reply:
No styling rules in ActionMailer :D